### DISCUSSION TITLE
Using Fluid with Web Components

### DISCUSSION THREAD

**Discussion by tylerbutler** [microsoft/FluidFramework#5465] [Documentation topics]
~This article isn't written yet, but you can discuss and vote on it here. We prioritize writing articles that receive the most votes.~

This article is now published here: <https://fluidframework.com/docs/recipes/web-components/>

**Comment by mhoad** [comment]
Would you be open to doing this with [Lit](http://lit.dev)?

**Reply by tylerbutler** [reply]
Yes! We don't have any expertise in Lit specifically, so I think we'd need some involvement from folks in the Lit community to make sure we "do it right." Are you offering? :)

**Comment by tylerbutler** [comment]
This article is now published here: <https://fluidframework.com/docs/recipes/web-components/>
